app/views/people/_relationship_action.haml

```diff
@@ -1,7 +1,8 @@
 - unless person == current_user.person
   - contact = current_user.contacts.find_by_person_id(person.id)
   - contact ||= Contact.new(:person => person)
-  = aspect_membership_dropdown(contact, person, 'right')
+  - gon_load_contact(contact)
+  .aspect_membership_dropdown.placeholder{data: {person_id: contact.person_id}}
```

Comment:
Looks like we have the same [method](https://github.com/diaspora/diaspora/blob/develop/app/controllers/people_controller.rb#L193) in controller that is called in order to generate arguments for the partial. The logic can be moved there.